Source/ACE/Entity/Player.cs

```diff
@@ -473,10 +259,32 @@ public void Load(AceCharacter character)
 
             IsOnline = true;
 
-            // Character.AnimationOverrides.ForEach(ao => this.ModelData.AddModel(ao.Index, ao.AnimationId));
-            // Character.TextureOverrides.ForEach(to => this.ModelData.AddTexture(to.Index, to.OldId, to.NewId));
-            // Character.PaletteOverrides.ForEach(po => this.ModelData.AddPalette(po.SubPaletteId, po.Offset, po.Length));
+            PhysicsData.MTableResourceId = Character.MotionTableId;
+            PhysicsData.Stable = Character.SoundTableId;
+            PhysicsData.Petable = Character.PhysicsTableId;
+            PhysicsData.CSetup = Character.ModelTableId;
+
+            ContainerCapacity = 7;
+
+            if (Character.DefaultScale != null)
+                PhysicsData.ObjScale = Character.DefaultScale;
+
+            // Hair/head
```

Comment:
Model data that cannot change easily (such as is defined at character creation) will probably need to ultimately go in a separate data structure so that it can be restored if a player takes off all equipment.